### PR TITLE
fixes

### PR DESCRIPTION
**Pull request by MuhammadKhalilzadeh**

### CODE REVIEW

**Review by ajhollid — APPROVED** [review]
This is a great PR! Less than 200 lines, focused on one feature, this is very easy to review.

Thank you for revising this!

File: Client/src/Pages/Maintenance/CreateMaintenanceWindow/index.jsx [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+const directory = {
+  title: "Create a maintenance window",
+  description: "Your pings won’t be sent in this time frame.",
+};
+
+const repeatOptions = [
+  { _id: 1, name: "Don't repeat" },
+  { _id: 2, name: "Repeat daily" },
+  { _id: 3, name: "Repeat weekly" },
+];
```
Comment (ajhollid) [inline Client/src/Pages/Maintenance/CreateMaintenanceWindow/index.jsx:22]: Nice, thanks for revising that

**Review by ajhollid — APPROVED** [review]

**Review by danielcj2 — APPROVED** [review]
Looking good! Definitely a big improvement, nice stuff 🙌

File: Client/src/Pages/Maintenance/CreateMaintenanceWindow/index.jsx (hunk 1 above)
Comment (MuhammadKhalilzadeh) [reply Client/src/Pages/Maintenance/CreateMaintenanceWindow/index.jsx:22]: 👍🏻🍻

**Pull request merged**